.github/helper/update_pot_file.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+set -e
+cd ~ || exit
+
+echo "Setting Up Bench..."
+
+pip install frappe-bench
+bench -v init frappe-bench --skip-assets --python "$(which python)" --frappe-path "${GITHUB_WORKSPACE}"
+cd ./frappe-bench || exit
+
+echo "Generating POT file..."
+bench generate-pot-file --app frappe
+
+echo "Commiting changes..."
+cd "${GITHUB_WORKSPACE}"
+git checkout -b update-pot-file
+git add .
+git commit -m "chore: update POT file"
+
+echo "Creating a PR..."
+gh pr create --base "${BRANCH}" --head update-pot-file
```

Comment:
doubt this will work :grimacing: 

We'll have to use something like https://github.com/marketplace/actions/create-pull-request

Reply:
This was the first thing I found in the official docs: https://docs.github.com/en/actions/using-workflows/using-github-cli-in-workflows

But yeah, the dedicated PR action seems more comfortable

Reply:
The script is now tested to work, apart from the final step, which still gives me "pull request create failed: GraphQL: Resource not accessible by integration (createPullRequest)". Couldn't figure out why yet (repo settings are configured to allow PR creation by actions).

Reply:
We have bot's token in secrets, can use that instead 👀